lib/commands/xctest.js

```diff
@@ -0,0 +1,87 @@
+import B from 'bluebird';
+import { logger } from 'appium-support';
+import log from '../logger';
+import appManagement from './app-management';
+
+let helpers = {}, commands = {}, extensions = {};
```

Comment:
we don't need helpers and extensions here

Reply:
I tried taking those out and it failed. I think it might be a duck-type pattern where every file imported from `commands/` is expected to export helpers, commands and extensions.

Reply:
it should simply work: https://github.com/appium/appium-xcuitest-driver/blob/d2d895be17bf89590fafdc66989054146885c0c9/lib/commands/recordscreen.js#L11

Reply:
I tried and now I'm getting `Original error: this[commandMap[mobileCommand]] is not a function`

Reply:
Let me try something...